### ISSUE TITLE
Pagination does not work with ElasticSearch version 7

### ISSUE THREAD

**Issue by paulorodrigues2** [searchkit/searchkit#696]
The title pretty much says it. I've been basically writing custom show last page link functions to make up for this. I know the docs say "often slow in ElasticSearch", but it just doesn't seem to be working at all.

Any help would be appreciated,
Thanks

**Comment by Fluorz** [comment]
:+1: Don't work for me too

**Comment by Fluorz** [comment]
I found a trick to make it work.

Pagination is not working because hitsCount is also not working on ElasticSearch 7.

If you want an easy solution to make it work, you can :

go to this file: **node_modules/searchkit/lib/core/SearchkitManager.js**

**Change this** 

```
    SearchkitManager.prototype.getHitsCount = function () {
        return get(this.results, ["hits", "total"], 0);
    };
```

**For** 

```
    SearchkitManager.prototype.getHitsCount = function () {
        return get(this.results, ["hits", "total", "value"], 0);
    };
```

**Comment by flts** [comment]
The most easiest way to make it work in Elasticsearch 7 is to add rest_total_hits_as_int=true as a request parameter in the search request. This will make the total value a integer again instead of an object type.

https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#_literal_hits_total_literal_is_now_an_object_in_the_search_response

**Comment by joemcelroy** [comment]
fixed in v2.4.1
